### PR TITLE
use new certs for service network and localhost

### PR DESCRIPTION
**Pull request by deads2k**
The load balancer has not yet been switched.

/assign @sanchezl

### CODE REVIEW

File: bindata/v3.11.0/kube-apiserver/defaultconfig.yaml [hunk 1]
```diff
@@ -96,9 +96,15 @@ servicesSubnet: 10.3.0.0/16 # ServiceCIDR
...
   bindNetwork: tcp4
+<<<<<<< HEAD
   certFile: /etc/kubernetes/static-pod-certs/secrets/serving-cert/tls.crt
   clientCA: /etc/kubernetes/static-pod-certs/configmaps/client-ca/ca-bundle.crt
   keyFile: /etc/kubernetes/static-pod-certs/secrets/serving-cert/tls.key
+=======
+  certFile: /etc/kubernetes/static-pod-resources/secrets/service-network-serving-certkey/tls.crt
+  clientCA: /etc/kubernetes/static-pod-resources/configmaps/client-ca/ca-bundle.crt
+  keyFile: /etc/kubernetes/static-pod-resources/secrets/service-network-serving-certkey/tls.key
+>>>>>>> f3d97c30... serve using SNI and rotated certificates
```
Comment (sdodson) [inline bindata/v3.11.0/kube-apiserver/defaultconfig.yaml]: merge conflict cruft?

**Comment by deads2k** [comment]
looks like it mostly worked.  I need to update some e2e tests.

**Comment by deads2k** [comment]
this moves us closer.  Starting here.

File: pkg/operator/targetconfigcontroller/targetconfigcontroller.go [hunk 2]
```diff
@@ -126,6 +126,7 @@ func (c TargetConfigController) sync() error {
 	}
 	requiredPaths := [][]string{
 		{"serviceAccountPublicKeyFiles"},
+		{"servingInfo", "namedCertificates"},
```
Comment (sttts) [inline pkg/operator/targetconfigcontroller/targetconfigcontroller.go]: this break upgrade. My cluster does not have this field.

File: bindata/bootkube/manifests/config.apiserver.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+apiVersion: config.openshift.io/v1
+kind: APIServer
+metadata:
+  name: cluster
```
Comment (sttts) [inline bindata/bootkube/manifests/config.apiserver.yaml]: this does not validate: spec is required

File: pkg/operator/targetconfigcontroller/targetconfigcontroller.go (hunk 2 above)
Comment (deads2k) [reply pkg/operator/targetconfigcontroller/targetconfigcontroller.go]: > this breaks upgrade. My cluster does not have this field.

The control loop for configobservation will add it as soon as it runs.

**Pull request merged**